.travis.yml

```diff
@@ -1,40 +1,51 @@
-language: php
+version: ~> 1.0
 
-dist: xenial
-
-services:
-  - mysql
-  - postgresql
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/self.yml
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=4.x-dev
-
-matrix:
+    - COMPOSER_ROOT_VERSION="4.7.x-dev"
+    - REQUIRE_EXTRA="silverstripe-themes/simple:~3.2.0 silverstripe/admin:1.7.x-dev silverstripe/versioned:1.7.x-dev mikey179/vfsstream:^1.6"
+    - PHPUNIT_SUITE="recipe-core"
+
+# Using a manual matrix to exlude PHPUNIT_COVERAGE_TEST which was having an unknown error
+# https://travis-ci.com/github/silverstripe/recipe-core/jobs/457587137
+jobs:
+  fast_finish: true
   include:
     - php: 7.1
-      env: DB=MYSQL COMPOSER_ARG=--prefer-lowest
```

Comment:
We should look at introducing a `--prefer-lowest` build to the shared matrix - it's a useful testing target. For now, it'd be good to retain it here.

Reply:
Have added it back in